application/src/test/java/com/sanction/thunder/email/EmailServiceTest.java

```diff
@@ -0,0 +1,64 @@
+package com.sanction.thunder.email;
+
+import com.amazonaws.AmazonClientException;
+import com.amazonaws.services.simpleemail.AmazonSimpleEmailService;
+import com.amazonaws.services.simpleemail.model.Body;
+import com.amazonaws.services.simpleemail.model.Content;
+import com.amazonaws.services.simpleemail.model.Destination;
+import com.amazonaws.services.simpleemail.model.Message;
+import com.amazonaws.services.simpleemail.model.SendEmailRequest;
+
+import com.amazonaws.services.simpleemail.model.SendEmailResult;
+import com.sanction.thunder.models.Email;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class EmailServiceTest {
+
+  private final AmazonSimpleEmailService emailService = mock(AmazonSimpleEmailService.class);
+  private final AmazonClientException clientException = mock(AmazonClientException.class);
+  private final SendEmailResult result = mock(SendEmailResult.class);
+
+  private final String from = "noreply@sanctionco.com";
+  private final String subjectString = "Account Verification";
+  private final String htmlBodyString = "HTML";
+  private final String bodyString = "TEXT";
```

Comment:
Can make these static and make the casing all caps. E.g. `private static final String BODY_STRING = "TEXT"`